tests/tls/tls-ja3s-requires/test.yaml

```diff
@@ -0,0 +1,22 @@
+pcap: ../../tls/tls-certs-alert/input.pcap
+
+requires:
+  min-version: 7.0.3
+  features:
+    - HAVE_LUA
+    - HAVE_JA3
+  script:
+    - grep 'ProvidesFeature(FEATURE_JA3);' src/app-layer-ssl.c > /dev/null
```

Comment:
Not sure about this precondition...

Why is it needed ?
The SV PR will only be merged after the Suricata one anyways, right ?

Reply:
Yes, but I also wanted the tests to also succeed in the PR here, where they are not necessarily tested against the branch of Suricata that this branch is targeted at. Some tests might not work correctly with the Suricata version used in the S-V CI.